### ISSUE TITLE
[Known Issue] 404 not found while creating or updating project

### ISSUE THREAD

**Issue by joshuadesierto** [octobercms/october#5529]
404 not found https://raw.githubusercontent.com/octoberrain/meta/master/manifest/builds.json while creating procject

**Comment by daftspunk** [comment]
This error is safe to ignore and will be fixed in a release tomorrow (1.1.2). Apologies for the inconvenience. Will pin this issue for others to see

**Comment by adrenth** [comment]
To be clear for everyone, this issue occurs when you run:

```
➜ php artisan october:version
*** Detecting October CMS build...

In SourceManifest.php line 72:
                                                                                                                                                                         
  file_get_contents(https://raw.githubusercontent.com/octoberrain/meta/master/manifest/builds.json): failed to open stream: HTTP request failed! HTTP/1.1 404 Not Found  
```

Or when running a `composer update` command.

**Comment by daftspunk** [comment]
:partying_face: Version 1.1.2 has been released with PHP 8.0 support :elephant: This will be the last version before the next big update. Enjoy! :blush:

Issue fixed as a result
